pydm/widgets/slider.py

```diff
@@ -481,7 +486,7 @@ def userMaximum(self, new_max):
         ----------
         new_max : float
         """
-        self._user_maximum = float(new_max)
+        self._user_maximum = float(new_max) if new_max is not None else None
```

Comment:
What is the consequence of None in here at `self._user_maximum`?

Reply:
In `reset_slider_limits`, if the minimum or maximum is none (from the user, or from the channel), it puts the widget into a 'needs limit info' state, which disables the widget.  The internal slider's minimum or maximum value just remain wherever they were (likely the default values) until the min/max gets set to a non-None value.

So, setting this to None should be fine.

Reply:
Awesome! Thank you for clarifying, @mattgibbs. 👍